openpype/hosts/maya/plugins/publish/collect_instances.py

```diff
@@ -108,17 +108,22 @@ def process(self, context):
                                        data["asset"])
 
             # Append start frame and end frame to label if present
-            if "frameStart" and "frameEnd" in data:
+            if "frameStart" in data and "frameEnd" in data:
                 # Take handles from context if not set locally on the instance
                 for key in ["handleStart", "handleEnd"]:
                     if key not in data:
                         data[key] = context.data[key]
```

Comment:
```suggestion
                        data[key] = int(context.data[key])
```
Not sure why other people are getting floats, but this could be a culprit.